### ISSUE TITLE
F5 BIG-IP VE image is successful patched, but VE cannot login

### ISSUE THREAD

**Issue by wynn5a** [F5Networks/f5-openstack-heat#156]
Hi All,

I've patched the VE image for openstack ready, but I cannot login system by root/default.

- Step1: use heat template for BIG_IP 12 image patching
- Step2: use uploaded ve image in openstack to  launch a BIG_IP VE instance
- Step3: login terminal using default password `default` for root
result: login incorrect

#### Environment information
- Openstack version: newton
- Heat plugin version : 
  ```
  f5-icontrol-rest==1.3.0
  f5-openstack-heat-plugins==10.0.0
  f5-sdk==2.3.3
  ```
- snapshot
![incorrect-login](https://user-images.githubusercontent.com/3407317/28413687-18e54fc2-6d7b-11e7-9b3d-de5b0b8573ae.png)

Thanks for your help

**Comment by ssorenso** [comment]
Hello @wynn5a,

Thank you for pointing out this bug to us!  I was able to re-create it.  The available 12.1.2 image is not able to be loaded with the root password set properly.  May I suggest 12.1.1 for the time being, or perhaps 13.0.0?  Both of these have been extensively tested, and I will review 12.1.2 and see if I cannot get an appropriate version put to that available hyperlink.

Thanks and sorry for the inconvenience.  Please let us know if there is anything further we can help you with.  Meanwhile, I will be looking into a root cause.

Cheers,
ssorenso

**Comment by wynn5a** [comment]
@ssorenso 
Thank you for your patience. It helps me a lot. 
I will redeploy BIG-IP following those steps.
Close for no bug exists.
